DependencyInjection/DoctrineExtension.php

```diff
@@ -394,18 +397,28 @@ protected function ormLoad(array $config, ContainerBuilder $container)
             $container->removeAlias('doctrine.orm.metadata.annotation_reader');
         }
 
+        // available in Symfony 6.3
+        class_exists(DoctrineDbalCacheAdapterSchemaListener::class) ?
```

Comment:
$container->removeDefinition('doctrine.orm.listeners.doctrine_dbal_cache_adapter_schema_'.(class_exists(DoctrineDbalCacheAdapterSchemaListener::class) ? 'listener' : 'subscriber));`
and let's do this also for pre-existing lines below

Reply:
ok ! if I'm not mistaken, there is just the case for the 3 new listeners. I'll double check